libsql-storage/src/lib.rs

```diff
@@ -74,13 +77,24 @@ impl WalManager for DurableWalManager {
     ) -> Result<Self::Wal> {
         let db_path = OsStr::from_bytes(&db_path.to_bytes());
         let namespace = self.resolver.resolve(db_path.as_ref());
+        let cache_path = {
+            let parent_dir = Path::new(db_path).parent().unwrap_or(Path::new("."));
+            &parent_dir.join(format!("{}_local_cache.db", namespace))
+        };
```

Comment:
currently, I am creating the cache file in the same location where db file exists.